lib/Connection.php

```diff
@@ -249,7 +256,7 @@ private function closeSocket() {
         $this->outputBuffer = "";
         $this->outputBufferLength = 0;
 
-        if (is_resource($this->socket)) {
+        if (!$this->persistent && is_resource($this->socket)) {
```

Comment:
I have test it, if the socket is closed, the persistent connection will be use in one request. We need use the connection in the children fpm process.